include/uk/plat/lcpu.h

```diff
@@ -60,22 +53,28 @@ void ukplat_lcpu_disable_irq(void);
 
 /**
  * Returns current interrupt flags and disables them
- * @return interrupt flags (Note that the format is unspecified)
+ *
+ * @return interrupt flags (the format is unspecified)
  */
 unsigned long ukplat_lcpu_save_irqf(void);
 
 /**
  * Loads interrupt flags
- * @param flags interrupt flags (Note that the format is unspecified)
+ *
+ * @param flags interrupt flags (the format is unspecified)
  */
 void ukplat_lcpu_restore_irqf(unsigned long flags);
 
 /**
  * Checks if interrupts are disabled
+ *
  * @return non-zero value if interrupts are disabled
  */
 int ukplat_lcpu_irqs_disabled(void);
 
+/**
```

Comment:
Looks like it's been used in lib/ukschedcoop/schedcoop.c

Reply:
Yes, but it seems the functionality to look for pending IRQs is only used in Xen. If I remember correctly I checked the code and thought this could also be done differently without having to introduce a public API that makes no sense for other platforms. But I would leave this as TODO for a future PR.